README.md

```diff
@@ -36,8 +36,9 @@ This is a Spring Boot Application.
 Make sure that you have installed locally:
 - Java 17+
 - Maven 3.9+
+- Google Cloud SDK (`gcloud` CLI) configured with your project
```

Comment:
Please move this line to section "## Deployment to Google Cloud Platform (GCP)".
Not all developers working on this project have to deploy an app